src/components/YourCompany/CompanyName/CorrectNameRequest.vue

```diff
@@ -2,7 +2,7 @@
   <v-form id="correct-nr-form" ref="correctNrForm" v-model="valid" lazy-validation>
     <v-layout row>
       <v-flex md1 class="pa-3 pl-0">
-        <v-btn x-small fab outlined disabled color="#1A5A96" class="step-icon">1</v-btn>
+        <v-btn x-small fab outlined :ripple="false" color="#495057" class="step-icon">1</v-btn>
```

Comment:
If this color was declared in the Vuetify styles then I think you could use its name here (which would be a good consistency improvement). Check it out.

Eg, https://github.com/bcgov/namerequest/blob/master/client/src/plugins/vuetify.ts

Reply:
For sure! In this PR i will update the Vuetify presets and this colour specifically. 
In a following ticket i'll be updated some styles more globally and can update other instances where a hex is used within the templates.

Reply:
Awesome. Just be aware this may change Vuetify elements with class "error" or "primary" or whatever.

I don't know the complete list properties that are available. See https://vuetifyjs.com/en/features/theme/#customizing.

Reply:
And this change is not worth a lot of effort IMHO.

Reply:
I really like this though! I'm not a fan of using the hex anywhere and the template hex was always a sore spot. So this is a cool improvement!

Just added in the two relevant colours for now but we can expand this as we go :)

Reply:
Yep. Nicely done! 👍